Dockerfile

```diff
@@ -9,8 +9,31 @@ RUN export DEBIAN_FRONTEND="noninteractive" && apt-get update && \
     apt-get install -y --no-install-recommends \
       kicad kicad-libraries zip inkscape make git libmagickwand-dev \
       python3 python3-pip python3-wheel python3-setuptools inkscape \
-      libgraphicsmagick1-dev libmagickcore-dev openscad
+      libgraphicsmagick1-dev libmagickcore-dev openscad && \
+      rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install Pcbdraw KiKit
+RUN pip3 install Pcbdraw
```

Comment:
I wouldn't separate PcbDraw installation. Also, I would consider specifying version constraints on the Python packages installed. Then the act of updating dependency triggers re-build. And the process is explicit.

Reply:
PcbDraw must've been a leftover. I'll update that.

I'm fine with locking down the versions if you prefer that. Should I nail them at the current installed versions?

Reply:
I would lock them to at least major versions. Not sure about the minor version (no preference here). I wouldn't lock them to the patch version.

Reply:
I now pinned the versions to compatible minor versions.